sdk/cognitiveservices/Vision.ComputerVision/src/Generated/ComputerVisionClient.cs

```diff
@@ -1195,7 +1195,7 @@ public async Task<HttpOperationResponse<DomainModelResults>> AnalyzeImageByDomai
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<HttpOperationResponse<OcrResult>> RecognizePrintedTextWithHttpMessagesAsync(bool detectOrientation, string url, OcrLanguages language = default(OcrLanguages), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<HttpOperationResponse<OcrResult>> RecognizePrintedTextWithHttpMessagesAsync(bool detectOrientation, string url, OcrLanguages? language = default(OcrLanguages?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
No swagger change for this API, so it seems that this change is caused by the autorest tool change.